scala-package/examples/src/test/scala/org/apache/mxnetexamples/infer/profiler/ProfilerSuite.scala

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
Why infer, it should live in `mxnetexamples` at least

Reply:
You're right. Stuck it in the wrong place.